app/models/conference_preference.rb

```diff
@@ -1,6 +1,5 @@
 class ConferencePreference < ActiveRecord::Base
-  validates :terms_accepted?, acceptance: true
-
+  validates :terms, acceptance: true
```

Comment:
With the type of validation you're applying now, I think this is no longer an "acceptance: true" validation. It is instead now a so called [custom method validation](http://guides.rubyonrails.org/active_record_validations.html#custom-methods).

Therefore, I'd also suggest to rename this, so that it makes more sense for someone not familiar with the domain looking at it for the first time. Maybe just like you had it before
```ruby
validates :terms_accepted?
```